pkg/cloudprovider/provider/kubevirt/provider.go

```diff
@@ -83,7 +84,9 @@ func (k *kubeVirtServer) ID() string {
 func (k *kubeVirtServer) Addresses() []string {
 	var addresses []string
 	for _, kvInterface := range k.vmi.Status.Interfaces {
-		addresses = append(addresses, kvInterface.IP)
+		if address := strings.Split(kvInterface.IP, "/")[0]; address != "" {
```

Comment:
This will panic if `kvInterface.IP` is an empty string or not?

Reply:
The `Split` function would return a lengthed-one slice which is an empty string :)